databricks/koalas/frame.py

```diff
@@ -1354,7 +1354,7 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True, index=Tr
 
     # TODO: enable doctests once we drop Spark 2.3.x (due to type coercion logic
     #  when creating arrays)
-    def transpose(self, limit: Optional[int] = 1000):
+    def transpose(self, limit: Optional[int] = get_option("compute.max_rows")):
```

Comment:
I think we can entirely remove this `limit` parameter for now to be consistent with other APIs.

Reply:
and don't forget to remove the documentation in its `Parameters`.

Reply:
Related changes can be easily found via:

1. View File
![Screen Shot 2019-08-30 at 11 00 18 AM](https://user-images.githubusercontent.com/6477701/63988147-c3214500-cb15-11e9-9ef4-0b863f74814e.png)

2. Blame
![Screen Shot 2019-08-30 at 11 00 24 AM](https://user-images.githubusercontent.com/6477701/63988151-c4eb0880-cb15-11e9-8434-422153bb8ad9.png)

3. Go to the change and click the commit
![Screen Shot 2019-08-30 at 11 00 46 AM](https://user-images.githubusercontent.com/6477701/63988152-c61c3580-cb15-11e9-8e27-3b2e91fa5b31.png)